dev/fattest.simplicity/src/componenttest/topology/database/container/DatabaseContainerType.java

```diff
@@ -12,48 +12,57 @@
 
 import java.lang.reflect.Constructor;
 
+import org.testcontainers.containers.Db2Container;
 import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MSSQLServerContainer;
+import org.testcontainers.containers.OracleContainer;
 import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.utility.DockerImageName;
 
 import com.ibm.websphere.simplicity.config.DataSourceProperties;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_db2_jcc;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_derby_client;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_derby_embedded;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_microsoft_sqlserver;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_oracle;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_postgresql;
 import com.ibm.websphere.simplicity.log.Log;
 
 /**
  * This is a current list of database testcontainers that are in the database rotation.
  */
 @SuppressWarnings("rawtypes")
 public enum DatabaseContainerType {
-    DB2("jcc.jar", "org.testcontainers.containers.Db2Container", "Properties_db2_jcc"),
-    Derby("derby.jar", "componenttest.topology.database.container.DerbyNoopContainer", "Properties_derby_embedded"),
-    DerbyClient("derbyclient.jar", "componenttest.topology.database.container.DerbyClientContainer", "Properties_derby_client"),
-    Oracle("ojdbc8_g.jar", "componenttest.topology.database.container.OracleContainer", "Properties_oracle"),
-    Postgres("postgresql.jar", "componenttest.topology.database.container.PostgreSQLContainer", "Properties_postgresql"),
-    SQLServer("mssql-jdbc.jar", "org.testcontainers.containers.MSSQLServerContainer", "Properties_microsoft_sqlserver");
+    DB2("jcc.jar", Db2Container.class.getCanonicalName(), Properties_db2_jcc.class, DockerImageName.parse("ibmcom/db2:11.5.0.0a")),
+    Derby("derby.jar", DerbyNoopContainer.class.getCanonicalName(), Properties_derby_embedded.class, DockerImageName.parse("")),
+    DerbyClient("derbyclient.jar", DerbyClientContainer.class.getCanonicalName(), Properties_derby_client.class, DockerImageName.parse("")),
+    Oracle("ojdbc8_g.jar", OracleContainer.class.getCanonicalName(), Properties_oracle.class, //
+           DockerImageName.parse("kyleaure/oracle-18.4.0-expanded:1.0.slim").asCompatibleSubstituteFor("gvenzl/oracle-xe")),
```

Comment:
I agree, but I would actually prefer other FATs use the image `kyleaure/oracle-18.4.0-expanded:1.0.slim` and not `kyleaure/oracle-18.4.0-expanded:1.0.full.ssl` because the difference in size is 6GB!  That is why this enum is public :D